src/main/java/bio/terra/cloudevents/GCSEvent.java

```diff
@@ -0,0 +1,324 @@
+package bio.terra.cloudevents;
+
+import java.util.Date;
+
+public class GCSEvent {
```

Comment:
I think we should just use Google's `StorageObjectData` class.  I'm not sure there's a need for this class if we switch away from using `BackgroundFunction`?

Reply:
Replaced code that references `GCSEvent` by `StorageObjectData`.